src/Hintman/Cli.hs

```diff
@@ -1,25 +1,24 @@
 module Hintman.Cli
-    ( Context(..)
-    , cliContext
+    ( CliArguments(..)
+    , cliArguments
     ) where
 
-import Options.Applicative (Parser, auto, execParser, fullDesc, help,
-                            helper, info, long, metavar, option, progDesc,
-                            short, showDefault, switch, value)
+import Options.Applicative (Parser, auto, execParser, fullDesc, help, helper, info, long, metavar,
+                            option, progDesc, short, showDefault, switch, value)
 
 
 -- | Represents the applicaiton settings
-data Context = Context
-    { contextLogging :: Bool
-    , contextPort :: Int
+data CliArguments = CliArguments
+    { argumentsLogging :: Bool
+    , argumentsPort    :: Int
```

Comment:
According to our style guide, name of the fields should start with full data type name. So this should be:

```haskell
cliArgumentsLogging :: Bool
```

Reply:
Thanks @ChShersh 

faf4e9a will fix it